mobile/components/Lists/ListItemWithAvatar.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import * as React from 'react';
+import TouchableOpacity from '../TouchableOpacity';
+import { Row } from '../Flex';
+import { ListItemView, AvatarWrapper } from './style';
+
+type ListItemWithAvatarProps = {
+  onPress: Function,
+  children?: React.ChildrenArray<any>,
+  AvatarComponent: React.ComponentType<any>,
+};
+
+export class ListItemWithAvatar extends React.Component<
```

Comment:
This is kind of a weird component, why does this need to exist? Surely users could just use the normal `ListItem` and render the avatar as a child?

```JS
<ListItem>
  <Avatar />
  <SomeOtherComponent />
</ListItem>
```

instead of the a bit awkward

```JS
<ListItemWithAvatar AvatarComponent={Avatar}>
  <SomeOtherThing />
</ListItemWithAvatar>
```